### ISSUE TITLE
Aluminum block crafts into unusable aluminum ingots

### ISSUE THREAD

**Issue by esotericist** [Y0ungSandwich/Mech-Magic-1.12.2#81]
**Describe the bug**
Crafting an aluminum block by itself produces Silent's Gems: Extra Parts aluminum ingots, which appears to not be usable for anything (including making an aluminum block again)

**Expected behavior**
I would expect to get usable aluminum ingots (presumably the ones from `OreGen`)

**Additional context**
In case it somehow matters, I got the aluminum block from casting into a tinkers' construct casting basin

**Comment by Y0ungSandwich** [comment]
Interesting. No real reason it shouldn't have auto converted to OreGen while in your inventory. Ill check it out.

**Comment by esotericist** [comment]
No hurries; it was SSP with cheats enabled, so I gave myself the right ingots to populate a compacting drawer, so I'm sorted myself. 

Just figured you'd like to know. :)

**Comment by Y0ungSandwich** [comment]
Odd. Everything I am trying results in OreGen blocks, ingots, etc. Even out of the smeltery I am getting OreGen Aluminum Block -> OreGen Aluminum Ingot. You are on the newest version of M&M, 3.1.3? Maybe a config from UniDict didn't unzip properly at install.

**Comment by azulskyknight** [comment]
I experienced this with M&M 3.1.4 (ATLauncher).  Aluminum exiting the Smeltery via ingot cast would become OreGen ingots.  Blocks exiting the smeltery via the block cast would be an OreGen block.  But shaped crafting to convert that block into an ingot would result in a Silent's Gems ingot.  This ingot wouldn't melt in the smeltery and we couldn't use it in any recipes.  But I was able to toss it into an Immersive Engineering metal press, and turn them into plates.  Those would melt in the Smeltery and once passed out the ingot cast they were fine to use.
